src/main/java/nextstep/session/domain/SessionCoverImage.java

```diff
@@ -0,0 +1,74 @@
+package nextstep.session.domain;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import nextstep.session.InvalidImageConditionsException;
+
+public class SessionCoverImage {
+
+    private final int MIN_WIDTH = 300;
+    private final int MIN_HEIGHT = 200;
+    private final int MAX_SIZE = 1024 * 1024;
+
+    private final long width;
+    private final long height;
```

Comment:
```suggestion
    private final long width;
    private final long height;
```

객체로 분리한다면 특별한 책임을 줄 수 있지 않을까요?